codalab/lib/cli_util.py

```diff
@@ -109,6 +109,11 @@ def get(dep):  # Return the key
                     'key %s exists with multiple values: %s and %s' % (key, key2val[key], val)
                 )
         else:
+            if key is None:
+                raise UsageError(
+                    'Dependency is empty. Please provide a valid dependency in the format of :<dependency>.'
```

Comment:
It's really should be '[<key>]:<dependency>' or something - can you use the actual format of a 'target' for consistency, especially, if you can just use it as a variable'...

Reply:
What do you mean by `[]:`?

Reply:
Oh, `[&lt;key&gt;]:`...(angle brackets got swallowed up)

Reply:
There should be a constant somewhere in our codebase that specifies the actual format. All the format strings like that are called `*_SPEC` and this one might be called `DEPENDENCY_SPEC` or something like that. It would be good to find and use that for consistency.

Reply:
Yeah